crates/blockifier/src/execution/contract_class.rs

```diff
@@ -157,8 +160,9 @@ impl Decode for ContractClassV0Inner {
         input: &mut I,
     ) -> Result<Self, parity_scale_codec::Error> {
         let res = <(Program, Vec<(EntryPointType, Vec<EntryPoint>)>)>::decode(input)?;
+        let entry_point_btree = <BTreeMap<EntryPointType, Vec<EntryPoint>>>::from_iter(res.1);
```

Comment:
This is not needed.
On l162 decode into a `<BTreeMap<EntryPointType, Vec<EntryPoint>>>` directly. And then you only have to iterate once in order to collect into the final hashmap